programs/keeper-client/KeeperClient.cpp

```diff
@@ -0,0 +1,239 @@
+
+    options.addOption(
+        Poco::Util::Option("connection-timeout", "", "set connection timeout in seconds. default 10s.")
+            .argument("connection-timeout")
+            .binding("connection-timeout"));
+
+    options.addOption(
+        Poco::Util::Option("session-timeout", "", "set session timeout in seconds. default 10s.")
+            .argument("session-timeout")
+            .binding("session-timeout"));
+
+    options.addOption(
+        Poco::Util::Option("operation-timeout", "", "set operation timeout in seconds. default 10s.")
+            .argument("operation-timeout")
+            .binding("operation-timeout"));
+
+    options.addOption(
+        Poco::Util::Option("history-file", "", "set path of history file. default `~/.keeper-client-history`")
+            .argument("history-file")
+            .binding("history-file"));
+}
+
+void KeeperClient::initialize(Poco::Util::Application & /* self */)
+{
+    loadCommands({
+        {"set", 2, [](KeeperClient * client, const std::vector<String> & args)
+         {
+             client->zookeeper->set(client->getAbsolutePath(args[1]), args[2]);
+         }},
+
+        {"create", 2, [](KeeperClient * client, const std::vector<String> & args)
+         {
+             client->zookeeper->create(client->getAbsolutePath(args[1]), args[2], zkutil::CreateMode::Persistent);
+         }},
+
+        {"get", 1, [](KeeperClient * client, const std::vector<String> & args)
+         {
+             std::cout << client->zookeeper->get(client->getAbsolutePath(args[1])) << "\n";
+         }},
+
+        {"ls", 0, [](KeeperClient * client, const std::vector<String> & /* args */)
+         {
+             auto children = client->zookeeper->getChildren(client->cwd);
+             for (auto & child : children)
+                 std::cout << child << " ";
+             std::cout << "\n";
+         }},
+
+        {"ls", 1, [](KeeperClient * client, const std::vector<String> & args)
+         {
+             auto children = client->zookeeper->getChildren(client->getAbsolutePath(args[1]));
+             for (auto & child : children)
+                 std::cout << child << " ";
+             std::cout << "\n";
+         }},
+
+        {"cd", 0, [](KeeperClient * /* client */, const std::vector<String> & /* args */)
+         {
+         }},
+
+        {"cd", 1, [](KeeperClient * client, const std::vector<String> & args)
+         {
+             auto new_path = client->getAbsolutePath(args[1]);
+             if (!client->zookeeper->exists(new_path))
+                 std::cerr << "Path " << new_path << " does not exists\n";
+             else
+                client->cwd = new_path;
+         }},
+
+        {"rm", 1, [](KeeperClient * client, const std::vector<String> & args)
+         {
+             client->zookeeper->remove(client->getAbsolutePath(args[1]));
+         }},
+
+        {"rmr", 1, [](KeeperClient * client, const std::vector<String> & args)
+         {
```

Comment:
Would be nice to add some confirmation message here, because this operation is very destructive.